solution_architecture.md

```diff
@@ -396,6 +396,8 @@ If the back end calls from the mobile applications cannot be spread as evenly as
 
 ## CROSS-BORDER INTEROPERABILITY
 
+**Important update on cross-border interoperability: The European Federation Gateway Service was shut down EU-wide in February 2023. There is no more exchange of diagnosis keys via the gateway.** The following paragraphs describe the implementation up to the date of the shutdown in Febraury 2023.
```

Comment:
"Febraury" is misspelled

Reply:
Fixed with [`6cb9294` (#946)](https://github.com/corona-warn-app/cwa-documentation/pull/946/commits/6cb9294449a3353efa0919955873a56d8e65151c).